package.json

```diff
@@ -171,6 +171,7 @@
     "react-copy-to-clipboard": "5.0.2",
     "react-css-themr": "2.1.2",
     "react-dom": "16.11.0",
+    "react-helmet": "5.2.1",
```

Comment:
Note sure we need a new dependency for this.

If you instead modify `main_window.html` to use `<title><%= htmlWebpackPlugin.options.title %></title>` and then modify the `HtmlWebpackPlugin` for the `main_window` to use `title: 'foo bar',` you can set the title at compile time

That being said the way you do it is more flexible and probably easier to read. It just adds in a new dependency.